src/forms/FormField.js

```diff
@@ -22,44 +23,52 @@ const FormField = ({
   children,
   readOnly,
   newStudy,
-}) => (
-  <Form.Field required={required}>
-    <label className="noMargin">{name}:</label>
-    {!readOnly && description && (
-      <p className="noMargin">
-        <small>{description}</small>
-      </p>
-    )}
-    {children ? (
-      children
-    ) : (
-      <Form.Input
-        className={readOnly ? 'readOnlyField noMargin' : 'noMargin'}
-        fluid
-        readOnly={readOnly}
-        type={type}
-        name={id}
-        placeholder={placeholder}
-        open={focused}
-        onChange={handleChange}
-        onBlur={e => {
-          handleFocus('');
-          handleBlur(e);
-        }}
-        onFocus={e => handleFocus(id)}
-        value={value}
-        error={
-          touched !== undefined && errors !== undefined && errors.length > 0
-        }
-      />
-    )}
-    {touched && errors && errors.length > 0 && (
-      <Label pointing basic color="red">
-        {errors}
-      </Label>
-    )}
-  </Form.Field>
-);
+  isAdmin,
+}) => {
+  const hasError = touched && errors && errors.length > 0;
+  const tracking =
```

Comment:
not sure what this does 🤷‍♂

Reply:
This will return true or false for us to show/hide the red corner label on each field. We want to check if the current user is ADMIN, if the field is one of the fields we want to track, and if it has already had value. I put it outside the component instead of inline to make the component looks cleaner.